utils/spdx/src/main/resources/licenserefs/LicenseRef-ort-oracle-futc

```diff
@@ -0,0 +1,65 @@
+License Rights and Restrictions
```

Comment:
At the reviewer: Please check if you agree with the part of the text I have taken from the source.

Reply:
I actually believe we should really take all of the text from that page, starting with the "Oracle Free Use Terms and Conditions" title. Esp. the definitions are required IMO, as the "License Rights and Restrictions" make use of the defined terms.

Reply:
I have added the full text.